templates/Manifest.json

```diff
@@ -3,12 +3,12 @@
   "name": "chef-client-{{ platform }}{{ suffix }}",
   "publisher": "chef-software",
   "version": "{{ version }}",
-  "displayName": "ms-resource:displayName",
-  "publisherDisplayName": "ms-resource:publisherDisplayName",
-  "publisherLegalName": "ms-resource:publisherDisplayName",
-  "summary": "ms-resource:summary",
-  "longSummary": "ms-resource:description",
-  "description": "ms-resource:description",
+  "displayName": "Chef Extension ({{ version }}{{ extra }} {{ platform }})",
```

Comment:
Can this be `"Chef VM Extension for {{ platform }}"` and can we move the version into the description field shown on the portal? Otherwise we get the build number for the UI shown and that has proven confusing because there's no correlation between that and the actual version of extension installed.

So it's like `"({{ version }}{{ extra }})\r\nInstall the Chef Infra Client on your VM and bootstrap to your Chef Server"`

Reply:
Done